apps/foosball-api/server/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:16-slim
+
+ARG NODE_ENV=production
+ENV NODE_ENV=${NODE_ENV}
+
+WORKDIR /app
+COPY package.json ./
+
+RUN npm install -only=production
```

Comment:
A few things;
- What is this `-only=productions` supposed to do exactly?
  - Should it have two dashes `--only`?
-  Depending on what's supposed to do, does this work as expected?
   - This argument is no where mentioned in the `npm install` [docs](https://docs.npmjs.com/cli/v8/commands/npm-install).
   - Should this just be `--production`? Which is obsolete since the `NODE_ENV` is already set to `production`?

Reply:
The intention of the command, is to only install production dependencies. But if it is obsolete, I will remove it anyways.

Reply:
Updated.

Reply:
As per the docs, I would say so yeah;
> With the `--production` flag (or when the `NODE_ENV` environment variable is set to production), npm will not install modules listed in devDependencies. To install all modules listed in both dependencies and devDependencies when `NODE_ENV` environment variable is set to production, you can use `--production=false`.